Uplift/Controllers/ClassListViewController.swift

```diff
@@ -149,11 +153,19 @@ class ClassListViewController: UIViewController {
         if classList[index].isEmpty {
             let dateFormatter = DateFormatter()
             dateFormatter.dateFormat = "yyyy-MM-dd"
+            
+            pendingNetworkRequests += 1
```

Comment:
does this only ever toggle between 0 and 1. if so, lets just switch it to a boolean `isRequestInProgress`

Reply:
It represents the total number of ongoing requests. I believe that view controller makes two requests, so there could potentially be two requests ongoing at once.